### DISCUSSION TITLE
Issue with using oh-my-zsh

### DISCUSSION THREAD

**Discussion by segevfiner** [mattmc3/antidote#187] [Q&A]
Even after following https://github.com/mattmc3/antidote/discussions/70 and trying belak/zsh-utils path:completion, I'm hitting a `command not found: compdef` error when loading `oh-my-zsh`...

I wonder what's the right way to handle this... `antigen` used to have a buggy feature (Which I had to fix and was never merged, as antigen is abandoned)  to defer `compdef` calls like this, since you often load plugins before `compinit` and some try to call `compdef` directly instead of adding completion files to the `fpath`. Doesn't look like `belak/zsh-utils path:completion` handles this, and loading oh-my-zsh after `compinit` is likely to not work right either as it does add stuff to `fpath` as well... So it's kind of a chicken and egg problem...

```
/root/.cache/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/lib/directories.zsh:34: command not found: compdef
/root/.cache/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/plugins/git/git.plugin.zsh:101: command not found: compdef
/root/.cache/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/plugins/git/git.plugin.zsh:182: command not found: compdef
/root/.cache/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/plugins/git/git.plugin.zsh:209: command not found: compdef
/root/.cache/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/plugins/git/git.plugin.zsh:216: command not found: compdef
/root/.cache/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/plugins/git/git.plugin.zsh:246: command not found: compdef
/root/.cache/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/plugins/git/git.plugin.zsh:272: command not found: compdef
/root/.cache/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/plugins/git/git.plugin.zsh:286: command not found: compdef
/root/.cache/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/plugins/git/git.plugin.zsh:297: command not found: compdef
/root/.cache/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/plugins/git/git.plugin.zsh:308: command not found: compdef
/root/.cache/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/plugins/git/git.plugin.zsh:327: command not found: compdef
```

**Comment by mattmc3** [comment]
Linking #186

**Reply by mattmc3** [reply]
This problem is solved by including the brand new [getantidote/use-omz](https://github.com/getantidote/use-omz) plugin at the top of your .zsh_plugins.txt file. Using this plugin is now the recommended way of using Oh-My-Zsh's subplugins a la carte with antidote. It's a lightweight plugin that solves dependency issues when using OMZ with antidote, and is now the recommended way to use OMZ with antidote. There's also [getantidote/use-prezto](https://github.com/getantidote/use-prezto) which does a similar dependency resolution for Prezto users.
